.travis.yml

```diff
@@ -1,5 +1,17 @@
 sudo: false
 
+git:
+  depth: 1
+
+branches:
+  only:
+    - master
```

Comment:
Using this make that you MUST do a PR in order to have test on Travis. So if you try things on a branch, no CI. Not sure if that's what your want in the first place (that's why you have travis/push (simple commit) and travis/pr (merge push in master).